easybuild/easyconfigs/m/MinPath/MinPath-1.6-intel-2021b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'Tarball'
+
+name = 'MinPath'
+version = '1.6'
+# no tagged commit for MinPath 1.6, commit was determined based on commit history
+local_commit = '46d3e81a4dca2310d558bea970bc002b15d44767'
+
+homepage = 'https://omics.informatics.indiana.edu/MinPath'
+description = """MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions
+ using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological
+ pathways for a query dataset."""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
+
+source_urls = ['https://github.com/mgtools/MinPath/archive/'],
```

Comment:
@ItIsI-Orient You have an extra comma (`,`) at the end here, which is causing trouble, it's making EasyBuild ignore the specified sourcr URL:

```
== 2022-01-14 00:33:36,762 easyblock.py:825 WARNING Source URL ['https://github.com/mgtools/MinPath/archive/'] is of unknown type, so ignoring it.
```

The extra comma is essentially equivalent to using this:

```python
source_urls = (['https://github.com/mgtools/MinPath/archive/'], )
```

EasyBuild iterates over `source_urls`, obtain `['https://github.com/mgtools/MinPath/archive/']` as first element (a list with a string value in it), and doesn't know what to do with that (it expects a string value, not a list).